### PR TITLE
Allow PHP 8.0

### PR DESCRIPTION
**Pull request by VitalyArt**

### CODE REVIEW

**Comment by 48d90782** [comment]
@VitalyArt You should also add new tests to the `bors.toml`.
@wolfy-j @SerafimArts Should we stop supporting the PHP 7.2 as it supported until 30.11.2020?

**Comment by VitalyArt** [comment]
@48d90782 done👍

File: .github/workflows/ci-build.yml [hunk 1]
```diff
@@ -9,7 +9,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        php: [7.2, 7.3, 7.4]
+        php: [7.2, 7.3, 7.4, 8.0]
```
Comment (48d90782) [inline .github/workflows/ci-build.yml:12]: @wolfy-j What about 7.2? I assume that we should remove PHP 7.2 support, shouldn't we?

File: composer.json [hunk 2]
```diff
@@ -14,7 +14,7 @@
     }
   ],
   "require": {
-    "php": "^7.2",
+    "php": "^7.2 || ^8.0",
```
Comment (48d90782) [inline composer.json:17]: @wolfy-j Also, if we remove PHP 7.2 support, here we should change 7.2 -> 7.3

File: .github/workflows/ci-build.yml (hunk 1 above)
Comment (wolfy-j) [reply .github/workflows/ci-build.yml:12]: Yeah, we can drop it from tests.

**Review by wolfy-j — APPROVED** [review]
Looks good, 7.2 changes are not critical. The rest is OK.

File: .github/workflows/ci-build.yml (hunk 1 above)
Comment (48d90782) [reply .github/workflows/ci-build.yml:12]: @VitalyArt Could you please remove PHP 7.2 everywhere from `CI`, `bors.toml` and `composer.json`?

Comment (VitalyArt) [reply .github/workflows/ci-build.yml]: @48d90782 done

**Review by 48d90782 — APPROVED** [review]
LGTM

File: bors.toml [hunk 3]
```diff
@@ -1,10 +1,10 @@
 status = [
-'Build (PHP 7.2, Go 1.15, OS ubuntu-latest)',
-'Build (PHP 7.2, Go 1.14, OS ubuntu-latest)',
 'Build (PHP 7.3, Go 1.15, OS ubuntu-latest)',
 'Build (PHP 7.3, Go 1.14, OS ubuntu-latest)',
 'Build (PHP 7.4, Go 1.15, OS ubuntu-latest)',
 'Build (PHP 7.4, Go 1.14, OS ubuntu-latest)',
+'Build (PHP 8.0, Go 1.15, OS ubuntu-latest)',
+'Build (PHP 8.0, Go 1.14, OS ubuntu-latest)',
```
Comment (48d90782) [inline bors.toml:7]:
```suggestion
'Build (PHP 8, Go 1.15, OS ubuntu-latest)',
'Build (PHP 8, Go 1.14, OS ubuntu-latest)',
```

Comment (48d90782) [inline bors.toml:7]:
```suggestion
'Build (PHP 8, Go 1.15, OS ubuntu-latest)',
'Build (PHP 8, Go 1.14, OS ubuntu-latest)',
```

Comment (48d90782) [reply bors.toml:7]: @VitalyArt could you please update `bors.toml` as I suggested? `8.0` -> `8`

**Comment by 48d90782** [comment]
Thanks @VitalyArt. Now in `master`. 1.9.0 release with PHP 8.0 expected to be by end of week.

**Pull request merged**